### PR TITLE
Fix a bunch of typos in commands in Core

### PR DESCRIPTION
**Pull request by Fabian-Evolved**
### Type

- [ ] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes

Recent commit "Core Cog Guide" had a few grammatical mistakes, and some identation mistakes, i've fixed those in this pull request.

### CODE REVIEW

**Comment by Fabian-Evolved** [comment]
I just noticed i forgot to put a title on the pull request, I hope that is not an issue.

If it is, let me know, and i'll open a new one as soon as possible (read: when i wake up later)

File: redbot/core/core_commands.py [hunk 1]
```diff
@@ -3969,7 +3969,7 @@ async def localblocklist_remove(
         self, ctx: commands.Context, *users_or_roles: Union[discord.Member, discord.Role, int]
     ):
         """
-        Removes user or role from blocklist.
+        Removes user or role from local blocklist.
```
Comment (Drapersniper) [inline redbot/core/core_commands.py:3972]: IIRC this was done on purpose as @Kowlin  wanted all mentions to the allowlist and denylist at a local level to be just "blocklist" and "allowlist" and references to the global one to be "global allowlist" and "global blocklist"

Comment (Dav-Git) [reply redbot/core/core_commands.py:3972]: Check above in line 388, there it is also referenced as "local" so consistency is definitely an issue.

Comment (Fabian-Evolved) [reply redbot/core/core_commands.py:3972]: > IIRC this was done on purpose as @Kowlin wanted all mentions to the allowlist and denylist at a local level to be just "blocklist" and "allowlist" and references to the global one to be "global allowlist" and "global blocklist"

Alright i will get rid of that, do i also remove the one from line 388? and i'll push that too.

Comment (Kowlin) [reply redbot/core/core_commands.py:3972]: Feel free and just leave it @Fabian-Evolved, that's something for me to do when I make a gigantic pass over all the wording later in the year.

What Draper wasn't aware of is that I'd do that stuff later on.

**Review by Flame442 — APPROVED** [review]
Thanks for the fixes!

**Pull request merged**